
<div flex fxLayout="row" fxLayoutAlign="center center">
  <div fxFlex="90">

    <h1>About the SMARTER Database</h1>

    <p>
      The SMARTER project site is a place where WP4 partners of the
      <a href="https://www.smarterproject.eu/">SMARTER project</a>
      can browse and access their data. This dataset is now open and available
      to the community.
    </p>

    <h2>The SMARTER Project</h2>

    <p>
      Small ruminant populations play a fundamental role in the livelihood and
      socio-economic well-being of human settlements, especially in marginal areas
      of Europe. Under-utilized sheep and goat breeds may be highly valuable in
      increasing the profitability of small ruminant farming in these marginal areas.
      These breeds are valuable because they have unique and often atypical
      genetic make-ups, making them potentially extraordinary resources for
      adaptation to harsh environments, resilience to farming
      conditions, resistance to biotic and abiotic stressors, and the production
      of high-quality animal products.
    </p>

    <p>
      In this context, WP4 aims to address knowledge gaps concerning
      genetic diversity in goats and sheep. Its objective is to identify genetic
      factors that contribute to their ability to adapt to climate conditions and,
      more broadly, to withstand environmental disturbances.
      Available datasets relevant to genomic characterization
      were identified, and comprehensive data on hardy and under-utilized
      breeds were collected, including genotypes, phenotypes, and environmental
      measurements associated with rearing conditions.
      To ensure consistency, the recording of this data was standardized: genotypes
      were converted to the same genome version and coding convention.
      Finally, a shared database was established, consolidating
      the wealth of knowledge on genetic variation in small ruminants,
      as a culmination of the project's efforts.
    </p>

    <p>
      Data are made available to the community using four different instruments:
    </p>

    <ul>
      <li>
        The <a href="https://github.com/cnr-ibba/SMARTER-database">SMARTER database</a>,
        which is a collection of tools and scripts to collect, standardize, and provide
        a collection of genotype data and metadata information
        on hardy goat and sheep populations by combining new and existing datasets.
        This project is better explained and documented in its
        <a href="https://smarter-database.readthedocs.io/en/latest/index.html">manual pages</a>.
      </li>
      <li>
        The <a href="https://webserver.ibba.cnr.it/smarter-api/docs/">SMARTER backend</a>,
        which is a REST API service developed on top of the <em>SMARTER database</em>
        providing methods to interact with and access SMARTER data using web resources.
        By using the <em>SMARTER backend</em>, you can develop software or scripts that access
        SMARTER data via the web without needing a local instance
        of the <em>SMARTER database</em>. An example of this is the
        <a href="https://github.com/cnr-ibba/r-smarter-api">smarterapi R package</a>,
        which allows the collection of SMARTER data using the <em>SMARTER backend</em>. More
        information can be found in the
        <a href="https://smarter-backend.readthedocs.io/en/latest/index.html">SMARTER backend documentation</a>
        and <a href="https://cnr-ibba.github.io/r-smarter-api/index.html">smarterapi manual pages</a>.
      </li>
      <li>
        The <a href="https://github.com/cnr-ibba/SMARTER-frontend">SMARTER frontend</a>
        (<a routerLink="/">this site</a>), which is a web interface to browse <em>SMARTER database</em>
        data relying on the <em>SMARTER backend</em>.
      </li>
      <li>
        The SMARTER <a href="ftp://webserver.ibba.cnr.it/smarter/">FTP site</a>, which collects
        the genotype files of the animals described in the <em>SMARTER database</em>.
      </li>
    </ul>

    <p>
      These instruments can be used to collect metadata and other sample information. Genotypes
      are available for both <em>Goat and Sheep species</em> as whole PLINK binary files for each
      of the supported assemblies. You can download the genotype files from the
      <a href="ftp://webserver.ibba.cnr.it/smarter/">FTP site</a> for the available assemblies.
      Users are required to identify their samples of interest using the
      <a href="https://github.com/cnr-ibba/SMARTER-frontend">SMARTER frontend</a>,
      the <a href="https://webserver.ibba.cnr.it/smarter-api/docs/">SMARTER backend</a>, or the
      <a href="https://github.com/cnr-ibba/r-smarter-api">smarterapi R package</a>.
      These are three different ways to access the same data: the first one is a web interface,
      the second one is a REST API service, and the third one is an R package to interact with the API service.
      Users can then subset the genotype files according to their needs. Additional filtering steps
      on the selected genotypes can be done to remove missing samples or variants.
    </p>

    <h2>About This Site</h2>

    <p>
      You can browse WP4 SMARTER data using the different tabs. Data like samples, variants,
      and breeds can be browsed by <em>species</em> (Goat, Sheep). You will find
      a button on the top of the page to select the data you want to browse.
      Data like datasets are available on the same page for both species. Here's
      a list of the site sections:
    </p>

    <ul>
      <li>
        <a routerLink="/breeds">Breeds</a>: navigate data by breeds. You can
        search for a specific breed or get general information about how many samples
        belong to the same breed. You can follow the links to retrieve all samples
        of a particular breed.
      </li>
      <li>
        <a routerLink="/datasets">Datasets</a>: navigate data by datasets. Browse
        data relying on the files submitted by the partners or by file content. For
        a <em>genotypes</em> dataset, you can follow the links to retrieve all samples
        belonging to the same dataset.
      </li>
      <li>
        <a routerLink="/samples">Samples</a>: navigate samples by species.
        You can filter samples by filling out the form fields.
      </li>
      <li>
        <a routerLink="/variants">Variants</a>: navigate the variants used
        to normalize the genotype files and collect information about their positions
        and genotype codings. You can browse variants by species and
        by assembly versions, and filter variants by filling out the form fields.
      </li>
    </ul>

  </div>
</div>
